lib/src/util/customPropTypes.ts

```diff
@@ -49,3 +49,21 @@ export const optionalPercentNumber: PropTypes.Validator<number> =
     }
     return requiredPercentNumberValidator(props, propName, componentName);
   };
+
+export const requiredValueNumber: PropTypes.Validator<number> =
+  function requiredValueNumberValidator(props: Props, propName: string, componentName: string): Error | null {
+    const value = props[propName];
+
+    if (props['maxVal']) {
+      const max = props['maxVal'];
+      if (typeof value === 'number' && value >= 0 && value <= max) {
+        return null;
+      }
+      const err = `${componentName}: prop "${propName}" must be a number between 0 and ${max} inclusive;
+      received ${value}`;
```

Comment:
If I'm not mistaken this actually encodes the newline and the indentation into the error string. We don't want that do we?

Reply:
You are correct, that does encode the newline and the indentation, I have fixed here 811b214b36b355e0386f216455feb945afaa4b4b